File: SupportDesk.Web/src/types/comment.ts
export interface CreateCommentRequest {
  content: string;
  ticketId: number;
}

export interface CommentResponse {
  id: number;
  content: string;
  createdAt: string;
  ticketId: number;
  userId: number;
}

export interface CommentDetailResponse {
  id: number;
  content: string;
  createdAt: string;
  user: {
    id: number;
    name: string;
    email: string;
    role: string;
    isActive: boolean;
  } | null;
}

File: SupportDesk.API/Models/Comment.cs
namespace SupportDesk.API.Models;

public class Comment
{
    public int Id { get; set; }
    public string Content { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
    public int TicketId { get; set; }
    public Ticket? Ticket { get; set; }

    public int UserId { get; set; }

    public User? User { get; set; }

}
